content-src/components/Search/Search.jsx

```diff
@@ -132,9 +133,8 @@ export class _Search extends React.PureComponent {
           id="searchSubmit"
           className="search-button"
           onClick={this.onSearchClick}
-          title={this.props.intl.formatMessage({id: "search_button"})}>
-          <span className="sr-only"><FormattedMessage id="search_button" /></span>
-        </button>
+          title={this.props.intl.formatMessage({id: "search_button"})}
+          aria-label={this.props.intl.formatMessage({id: "search_button"})} />
```

Comment:
The `button` with `title` seems to be good in devtools, so no `aria-label` needed here

Reply:
I chose to double up title and aria-label wherever there was already a title because a11y docs seem to show that different screen readers will use different things - and it doesn't seem like having them both duplicates anything in the DOM. I can always change it though!

https://www.deque.com/blog/text-links-practices-screen-readers/

Reply:
Thanks. The comparison table:

D. Both Aria-label and title present besides anchor text
Aria-label + title
Anchor text + title
Anchor text + aria-label
Aria-label + title

So with a duplicate title and aria-label, some screen readers will just say the same thing twice? Although arguably maybe we should be setting them to be different things ?

Although looks like VO on OSX just doesn't read `title`s at all so it alone wouldn't be sufficient anyway.

Reply:
Yeah - I wasn't sure which case we wanted to optimize for, so I went for overkill :P The tricky thing is the cases where we have buttons that are icons only, or are hidden until focused, so we have no anchor text to be read - I wanted to make sure there was always something that would be picked up.